### PR TITLE
feat: major updates to cli and local pipeline

### PR DESCRIPTION
**Pull request by grusell**
* Pipeline and encoding-profile can now be inlined in job definition file
* Non-ABR encoding, i.e. CRF or constant QP can now be used
* VMAF results can be exported to CSV file
* VMAF analysis can be run without transcoding if transcoding has allready been done
* CPU time used for encoding is measured

### CODE REVIEW

File: src/suggest-ladder.ts [hunk 1]
```diff
@@ -16,32 +20,33 @@ export default async function suggestLadder(
...
-  let counter = 1;
-  vmafs.forEach(({ filename, vmaf }) => {
-    const [resolutionStr, bitrateStr] = filename.split('_');
-    const [widthStr, heightStr] = resolutionStr.split('x');
+  // logger.info('Loading VMAF data...');
+  // const vmafs = await getVmaf(directoryWithVmafFiles, onProgress);
+  // let counter = 1;
+  // vmafs.forEach(({ filename, vmaf }) => {
+  //   const [resolutionStr, bitrateStr] = filename.split('_');
+  //   const [widthStr, heightStr] = resolutionStr.split('x');
```
Comment (oscnord) [inline src/suggest-ladder.ts:33]: Can this be removed?

File: src/suggest-ladder.ts [hunk 2]
```diff
@@ -16,32 +20,33 @@ export default async function suggestLadder(
...
+  //       pairs.set(bitrate, [{ resolution: { width, height }, vmaf }]);
+  //     }
+  //   }
 
-    logger.info(`Finished loading VMAF ${counter}/${vmafs.length}.`);
-    counter += 1;
-  });
+  //   logger.info(`Finished loading VMAF ${counter}/${vmafs.length}.`);
+  //   counter += 1;
+  // });
```
Comment (oscnord) [inline src/suggest-ladder.ts:49]: Same as above. Maybe better to remove it than leave dead code?

**Review by oscnord — APPROVED** [review]
Just one comment but otherwise LGTM!

File: src/suggest-ladder.ts (hunk 1 above)
Comment (grusell) [reply src/suggest-ladder.ts]: fixed

File: src/suggest-ladder.ts (hunk 2 above)
Comment (grusell) [reply src/suggest-ladder.ts]: fixed

**Pull request merged**